diff --git a/src/dials/algorithms/merging/french_wilson.py b/src/dials/algorithms/merging/french_wilson.py
index 2e8756213..cb7736a16 100644
--- a/src/dials/algorithms/merging/french_wilson.py
+++ b/src/dials/algorithms/merging/french_wilson.py
@@ -110,7 +110,7 @@ def determine_binning_counting_sorted(x: np.ndarray, max_bins=60, min_bin_size=2
     n_points = len(x)
     points_per_bin = max(len(x) // max_bins, min_bin_size)
     n_bins = len(x) // points_per_bin
-    logger.debug(f"{n_bins=}, {points_per_bin=}")
+    logger.debug(f"{n_bins}, {points_per_bin}")
     return np.interp(
         np.linspace(0, n_points, n_bins + 1), np.arange(n_points), np.sort(x)
     )
