diff --git a/src/dxtbx/format/FormatNXmxDLS.py b/src/dxtbx/format/FormatNXmxDLS.py
index 842907ef..3a237375 100644
--- a/src/dxtbx/format/FormatNXmxDLS.py
+++ b/src/dxtbx/format/FormatNXmxDLS.py
@@ -2,7 +2,6 @@ from __future__ import annotations
 
 import datetime
 import os
-from functools import cached_property
 from pathlib import Path
 from typing import Union
 
@@ -79,7 +78,7 @@ class FormatNXmxDLS(FormatNXmx):
             except Exception:
                 self._bit_depth_readout = 16
 
-    @cached_property
+    @property
     def _meta(self):
         return find_meta_filename(self._image_file)
 
