diff --git a/setup.cfg b/setup.cfg
index 4b97dcc9..aa0577d1 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -14,6 +14,7 @@ classifiers =
     Natural Language :: English
     Operating System :: OS Independent
     Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
     Programming Language :: Python :: Implementation :: CPython
@@ -34,7 +35,7 @@ install_requires =
 packages = find:
 package_dir =
     =src
-python_requires = >=3.8
+python_requires = >=3.7
 zip_safe = False
 
 [options.packages.find]
diff --git a/setup.py b/setup.py
index 487b3dab..3a4e5946 100644
--- a/setup.py
+++ b/setup.py
@@ -106,7 +106,8 @@ def get_version() -> str:
     if "dev" not in __version_tag__:
         return __version_tag__
     # If we're in a git repository, then use "git describe"
-    if (git_ver := get_git_revision()) is not None:
+    git_ver = get_git_revision()
+    if git_ver is not None:
         return git_ver
     # If all else fails, return our development tag as-is
     return __version_tag__
diff --git a/src/xia2/Applications/xia2setup.py b/src/xia2/Applications/xia2setup.py
index eb92bfde..13b67717 100644
--- a/src/xia2/Applications/xia2setup.py
+++ b/src/xia2/Applications/xia2setup.py
@@ -290,7 +290,8 @@ def _filter_aliased_hdf5_sweeps(sweeps: list[str]) -> set[str]:
     hdf5_sweeps: dict[frozenset[str], str] = {}
 
     for s in sweeps:
-        if not is_hdf5_name(s) or not (filenames := _linked_hdf5_data_files(s)):
+        filenames = _linked_hdf5_data_files(s)
+        if not is_hdf5_name(s) or not filenames:
             deduplicated.add(s)
         elif filenames in hdf5_sweeps:
             # Bias in favour of using _master.h5 in place of .nxs, because of XDS
diff --git a/src/xia2/XIA2Version.py b/src/xia2/XIA2Version.py
index dba4a178..1182f416 100644
--- a/src/xia2/XIA2Version.py
+++ b/src/xia2/XIA2Version.py
@@ -1,7 +1,8 @@
 from __future__ import annotations
 
-from importlib import metadata
+# from importlib import metadata
 
-VersionNumber = metadata.version("xia2")
+# VersionNumber = metadata.version("xia2")
+VersionNumber = '3.13.0'
 Version = f"XIA2 {VersionNumber}"
 Directory = f"xia2-{VersionNumber}"
